Assets/Scripts/Waypoint Scripts/Waypoint.cs

```diff
@@ -61,7 +60,7 @@ public void UpdateLineColliders()
 
         public void UpdateLocation(Vector3 newLocation)
         {
-            Debug.Log("Updating Location");
+            Debug.Log("Peru TODO: IF this doesn't update the location of the waypoint what does?");
```

Comment:
seems like a pretty important todo @PeruDayani

Reply:
@apollovision

Reply:
We will clean up the waypoint logic itself in the next pull request. Currently, the position updating is happening through VRTK and we need to clean it up to be consistent throughout